paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -690,8 +702,9 @@ class ConvMKLDNNGradOpKernel : public paddle::framework::OpKernel<T> {
     // as well as attributes of primitive to be created
     // This name will be used as key when saving info into device context
     const std::string key = platform::ConvMKLDNNHandler::GetHash(
-        src_tz, weights_tz, fuse_relu, fuse_brelu, strides, paddings, dilations,
-        groups, ctx.op().Input("Input") + ctx.op().Input("Filter"));
+        src_tz, weights_tz, fuse_relu, fuse_leaky_relu, fuse_brelu, strides,
```

Comment:
Fuses are available for Inference only, so technically speaking fuse_leaky_relu should always be "false" here , hence no point of reading and declaring it here.

Reply:
Will take this into account in final solution, thanks!